less/index.less

```diff
@@ -47,13 +48,144 @@ table {
 	border-collapse: collapse;
 	border-spacing: 0;
 }
+/* Reset End */
 
+/* Less Start */
 /* Set every element's box-sizing to border-box */
+
+@batblue: rgb(10,145,191);
```

Comment:
good job with having these two variables, I would only add that you might want to put a comment above saying these are variable and what they should be doing.